templates/module/helfi_ahjo/paragraph--sote-section.html.twig

```diff
@@ -0,0 +1,34 @@
+{% macro menu_links(menu_tree, current_depth, max_depth, collapsible) %}
+  {% import _self as macros %}
+  {% for item in menu_tree %}
+    {%
+      set liClass = [
+      item.subitem and current_depth < max_depth ? 'menu-item menu-item--expanded block-taxonomymenu__tree-item block-taxonomymenu__tree-item--expanded' : 'menu-item block-taxonomymenu__tree-item',
```

Comment:
let's not use the classes from menu component, not to create potential conflicts